Pandas Basics/Pandas_basics.ipynb

```diff
@@ -0,0 +1,267 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Pandas_basics.ipynb",
+      "provenance": [],
+      "collapsed_sections": [
+        "SXNvQRJmkg7C"
+      ]
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "SXNvQRJmkg7C"
+      },
+      "source": [
+        "# Pandas DataFrames"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "E8xUMesmkqck"
+      },
+      "source": [
+        "Pandas is a high-level data manipulation tool developed by Wes McKinney. It is built on the Numpy package and its key data structure is called the DataFrame."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "cG-b90nWkdaK",
+        "outputId": "de28c3e4-2cff-4851-caaf-1f51dd7c03a3",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 118
+        }
+      },
+      "source": [
+        "# Q: Create a table listing: Country, Capital, Area, Population\n",
```

Comment:
Looks better, can you also copy and paste this line like in between all code maybe like for this one, between line 45 and 46? Similarly for **others too.**

Reply:
Yes, sure!
That will be easy.

And yes, spell check on the label: hacktoberfest-accepted